roles/config-idm-server/tasks/configure_idm.yml

```diff
@@ -1,38 +1,33 @@
 ---
```

Comment:
This file looks out-of-date ... please `rebase` with the latest master.

Reply:
Rebased to latest master upstream.